match/lib/match/runner.rb

```diff
@@ -133,13 +136,48 @@ def update_optional_values_depending_on_storage_type(params)
       end
     end
 
-    def fetch_certificate(params: nil, working_directory: nil, specific_cert_type: nil)
+    RENEWABLE_CERT_TYPES = [:mac_installer_distribution, :development, :distribution, :enterprise]
```

Comment:
maybe we should rename this to `RENEWABLE_CERT_TYPES_VIA_API`?

Reply:
Yeah, it's for sure will improve readability. Done in eb0db7b98482fa99ab595ebd6c3f7d0d9c8c390c.